src/serverdlg.cpp

```diff
@@ -115,10 +118,9 @@ CServerDlg::CServerDlg ( CServer* pNServP, CServerSettings* pNSetP, const bool b
     cbxLocationCountry->setAccessibleName ( tr ( "Combo box for location of this server" ) );
 
     // enable recorder
-    chbEnableRecorder->setAccessibleName ( tr ( "Checkbox to turn on or off server recording" ) );
-    chbEnableRecorder->setWhatsThis ( "<b>" + tr ( "Enable Recorder" ) + ":</b> " +
-                                      tr ( "Checked when the recorder is enabled, otherwise unchecked. "
-                                           "The recorder will run when a session is in progress, if (set up correctly and) enabled." ) );
+    chbJamRecorder->setAccessibleName ( tr ( "Checkbox to turn on or off server recording" ) );
+    chbJamRecorder->setWhatsThis ( "<b>" + tr ( "Jam Recorder" ) + ":</b> " +
+                                   tr ( "The recorder will run when a session is in progress, if (set up correctly and) checked." ) );
```

Comment:
```suggestion
                                   tr ( "If checked, the recorder will run when a session is in progress, if recording is set up correctly." ) );
```

Reply:
Maybe this way around?
```suggestion
                                   tr ( "If checked and set up correctly, the recorder will run when a session is in progress." ) );
```